src/Modules/Profile/Commands.Profile/InvokePowerBIRestMethod.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Net.Http;
+using System.Runtime.Serialization.Json;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.PowerBI.Commands.Common;
+using Microsoft.PowerBI.Common.Abstractions.Interfaces;
+
+namespace Microsoft.PowerBI.Commands.Profile
+{
+    [Cmdlet(CmdletVerb, CmdletName)]
+    [OutputType(typeof(string))]
+    public class InvokePowerBIRestMethod : PowerBICmdlet
```

Comment:
Are we able to relatively easily add E2E tests for this cmdlet?

Reply:
With some refactoring yes.